tez-runtime-library/src/main/java/org/apache/tez/runtime/library/common/shuffle/Fetcher.java

```diff
@@ -544,8 +545,8 @@ private HostFetchResult setupConnection(Collection<InputAttemptIdentifier> attem
       } else {
         InputAttemptIdentifier firstAttempt = attempts.iterator().next();
         LOG.warn(String.format(
-            "Fetch Failure while connecting from %s to: %s:%d, attempt: %s Informing ShuffleManager: ",
-            localHostname, host, port, firstAttempt), e);
+            "Fetch Failure while connecting from %s to: %s:%d, attempt: %s, url: %s Informing ShuffleManager",
+            localHostname, host, port, firstAttempt, baseURI.toString()), e);
```

Comment:
Do we need to call toString() here, %s can take any type & the output is always string right? BTW why aren't we using Logger format here like {} instead of String.format?

Reply:
using {} format would make sense but we cannot mix the format arguments and exception in any of the LOG.warn methods
toString can be removed yeah